src/components/Slider/SliderBase/SliderFilledTrack.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { bem } from "../SliderHelpers";
+
+function defineFilledTrackProps({ dimension, offset, reverse }) {
```

Comment:
if this is used just for style maybe reconsider "flattening" the object a bit and just return the style properties ?